packages/front-end/components/Modal.tsx

```diff
@@ -11,6 +11,8 @@ type ModalProps = {
   cta?: string;
   closeCta?: string;
   ctaEnabled?: boolean;
+  secondaryAction?: string;
+  secondaryActionClick?: () => void;
```

Comment:
Both of these need to be set for the secondary button to work.  Instead of two separate optional props, we can use one object prop with two required fields.

```
secondaryAction?: {
  display: string;
  onClick: () => void;
}
```

That way it's not possible to forget to define one of them and it gives a natural place to allow customizing color, etc.